requirements.txt

```diff
@@ -0,0 +1 @@
+dist/janusgraph_python-0.0.9.tar.gz
```

Comment:
is this file `requirements.txt` needed?

Reply:
@pluradj : This was needed so that I could host a preview of docs on (readthedocs)[https://janusgraph-python.readthedocs.io/en/latest/] . I'll me cleaning this once we are in final process for merge request.

Reply:
Yes, once we have approavals from all commiters, I can then go ahead and remove the requirements.txt as well as the directory containing that .tar.gz file

Reply:
Not an issue. This was primarily required so as to host the docs on public website for people to view it. But since a lot / those who needed to view must have already viewed it, I'm going to remove that in next commit.